modules/ROOT/examples/live-demos/revisionhistory/index.js

```diff
@@ -139,9 +139,7 @@ const revisions = [
 
 const revisionhistory_fetch = () => new Promise((resolve) => {
   setTimeout(() => {
-    const sortedRevisions = revisions
-      .sort((a, b) => new Date(a.createdAt) < new Date(b.createdAt) ? 1 : -1);
-    resolve(sortedRevisions);
+    resolve(revisions);
```

Comment:
Having a sort is fine, it was just implemented backwards.
```suggestion
    const sortedRevisions = revisions
      .sort((a, b) => new Date(a.createdAt) < new Date(b.createdAt) ? -1 : 1);
    resolve(sortedRevisions);
```